lib/runtime/wasmer/instance.go

```diff
@@ -29,10 +29,7 @@ import (
 // Name represents the name of the interpreter
 const Name = "wasmer"
 
-var _ runtime.LegacyInstance = (*LegacyInstance)(nil)
```

Comment:
why is this removed? could you re-add it?

Reply:
Does that have any effect?

Reply:
Oh, sorry. Had to learn that these are interface checks. I added it back to the code.